### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by lssaleman**
UI update

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -36,3 +36,47 @@
...
+  transition: background-color 0.3s ease, transform 0.2s ease;
+  cursor: pointer;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.2);
+  margin-bottom: 8px;
+  min-width: 300px;
+  width: 100%;
+}
+
+.button:hover {
+  background-color: #45a049;
```
Comment (alvaroaleman) [inline src/App.css]: The button ends up on top of the textbox when hovering and not otherwise. Can that be avoided?

Comment (lssaleman) [reply src/App.css]: Could you provide a screenshot please? That error does not seem to show up for me. I can also just remove the button hover option

Comment (alvaroaleman) [reply src/App.css]: Here:
image

Comment (lssaleman) [reply src/App.css]: I just removed the effect now since it isn't necessary

Comment (alvaroaleman) [reply src/App.css]: Thanks!

**Pull request merged**